dashboard/app/controllers/code_reviews_controller.rb

```diff
@@ -0,0 +1,62 @@
+class CodeReviewsController < ApplicationController
+  before_action :authenticate_user!
+
+  # Make sure we don't forget to authorize each action
+  check_authorization except: [:index]
+
+  # GET /code_reviews
+  # Returns the list of code reviews and associated comments for the signed-in
+  # user and the given script and level.
+  def index
+    params.require([:scriptId, :levelId])
+
+    code_reviews = CodeReview.get_all(
+      user_id: current_user.id,
+      script_id: params[:scriptId],
+      level_id: params[:levelId]
+    )
```

Comment:
These should also be sorted by most recent last

Reply:
Oh, I thought we decided that the ordering here wasn't important since they'll be sorted on the front-end?

Reply:
Oh right, I wasn't sure if we were going to do both. Feel free to leave as-is! That makes sense